src/components/fields/AddressField.js

```diff
@@ -18,28 +18,28 @@ const AddressInput = styled(Input).attrs({
   size: 'large',
   spellCheck: false,
   weight: 'bold',
-})`
-  ${android && 'height: 56;'}
-  flex-grow: 1;
-  margin-top: 1;
-  z-index: 1;
-`;
+})({
+  ...(android ? { height: 56 } : {}),
```

Comment:
Here (and in many other places), I'd prefer:
```suggestion
  ...(android && { height: 56 }),
```

I will not post individual comments for this. What's the reason for preferring this over that?